autoPyTorch/evaluation/train_evaluator.py

```diff
@@ -34,12 +34,74 @@ def _get_y_array(y: np.ndarray, task_type: int) -> np.ndarray:
 
 
 class TrainEvaluator(AbstractEvaluator):
+    """
+    This class builds a pipeline using the provided configuration.
+    Such configuration is fitted using the datamanager object retrieved from disc,
```

Comment:
can we say, This pipeline is fitted ... and then After the pipeline is fitted ...